kedro-datasets/kedro_datasets/ibis/table_dataset.py

```diff
@@ -107,7 +112,9 @@ def __init__(  # noqa: PLR0913
                 `create_{materialized}` method. By default, ``ir.Table``
                 objects are materialized as views. To save a table using
                 a different materialization strategy, supply a value for
-                `materialized` in `save_args`.
+                `materialized` in `save_args`. You can also include a
+                `mode` parameter ("append", "overwrite", "error", "ignore")
```

Comment:
It would be nice to explain these, as done in https://spark.apache.org/docs/latest/api/python/reference/pyspark.sql/api/pyspark.sql.DataFrameWriter.mode.html#pyspark.sql.DataFrameWriter.mode.

I wonder if "append" or "insert" makes more sense (or if should just support both). As I think about it more, maybe "append" is good, because it's clear it doesn't overwrite (e.g. [dbt supports "append" and "insert_overwrite", among other things](https://docs.getdbt.com/docs/build/incremental-strategy)).